caas/kubernetes/provider/application/constraints.go

```diff
@@ -109,6 +113,51 @@ func ApplyConstraints(pod *core.PodSpec, appName string, cons constraints.Value,
 	return nil
 }
 
+// ApplyCharmConstraints applies the specified charm constraints to the pod.
```

Comment:
This comment needs to be updated. Implying the pod would mean we are updating every container in the pod when we are really just going to modify the first container that has a name of `constants.ApplicationCharmContainer`.

Reply:
Yup you're right, let me change this and thanks for spotting this